src/components/MiniPaper.vue

```diff
@@ -26,9 +26,6 @@ export default {
     },
   },
   computed: {
-    visibilityClass() {
-      return this.isOpen ? 'visible' : 'invisible';
```

Comment:
The use of 'invisible' was stopping the transition from working in fading the mini-map out. I experimented and it seemed unnecessary so I removed it. I also though that having the method here didn't add much so I inlined it.